CampaignConfig.json

```diff
@@ -0,0 +1,126 @@
+{
+  "Please note": "<true/false>, for native SDK build.  True for release build, false for beta build",
```

Comment:
This is a new file included as a part of the new plugin, it's basically an additional config that regenerates your project when you re-install.